### PR TITLE
3.0 controller split

### PR DESCRIPTION
**Pull request by rchavik**
Split CroogoAppController into two separate controllers:
- `Croogo\Core\Controller\Controller`
- `Croogo\Core\Controller\Admin\Controller`

### CODE REVIEW

**Comment by Marlinc** [comment]
Wow very nice. I haven't even seen this for some reason. Would it be possible to get the conflicts resolved?

**Comment by rchavik** [comment]
I've rebased on top 3.0 now. Still lots of adjustments needed though.  We could merge it and slowly renames/complete the rest.

Your call.

**Comment by Marlinc** [comment]
Thanks a lot! I'd like to merge this pretty early on. The other stuff can be fixed later.

File: Blocks/src/Controller/Admin/Controller.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Blocks Admin Controller
+ *
+ * @category Blocks.Controller
+ * @package  Croogo.Blocks
+ * @since  1.5
+ * @author   Fahad Ibnay Heylaal <contact@fahad19.com>
+ * @license  http://www.opensource.org/licenses/mit-license.php The MIT License
+ * @link     http://www.croogo.org
+ */
+class Controller extends CroogoController {
```
Comment (Marlinc) [inline Blocks/src/Controller/Admin/Controller.php]: Would it make sense to name it ```ÀppController``` which is conventional for CakePHP plugins? http://book.cakephp.org/3.0/en/plugins.html#plugin-controllers

**Pull request merged**